src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/ApplyChangesToWorkspaceContext.cs

```diff
@@ -147,15 +147,19 @@ private void ProcessProjectBuildFailure(IProjectRuleSnapshot snapshot)
             //
             // We still forward those 'removes' of references, sources, etc onto Roslyn to avoid duplicate/incorrect results when the next
             // successful build occurs, because it will be diff between it and this failed build.
-            _context.LastDesignTimeBuildSucceeded = snapshot.IsEvaluationSucceeded();
+            if (_context != null)
```

Comment:
I assume all this is rewritten in @davkean 's language service branch.  But it feels like we would want to throw something in this case